src/background.ts

```diff
@@ -217,3 +217,9 @@ window.tri = Object.assign(window.tri || Object.create(null), {
 omnibox.init()
 
 // }}}
+
+// {{{ Obey Mozilla's orders https://github.com/tridactyl/tridactyl/issues/1800
+
+native.unfixamo();
```

Comment:
Are you sure this actually works? I think @cmcaine had a similar approach but struggled to get it working reliably.

Reply:
Works fine on my machine and with multiple profiles. @cmcaine what issues did you encounter?